### PR TITLE
Code changes merget from my `local` branch

### PR DESCRIPTION
**Pull request by ledvinap**
I have merged some changes from my local clone. It should be `mostly harmless`.

I'am not sure about typedef struct names ...but unifying naming conventions is probably useful ..

I can remove some commits if necessary or squash them ..

### CODE REVIEW

File: src/main/mw.c [hunk 1]
```diff
@@ -236,14 +237,14 @@ void annexCode(void)
     }
 
     if (feature(FEATURE_VBAT)) {
-        if ((int32_t)(currentTime - vbatLastServiced) >= VBATINTERVAL) {
+        if (cmp32(currentTime, vbatLastServiced) >= VBATINTERVAL) {
```
Comment (tracernz) [inline src/main/mw.c]: Unfortunately this looks a lot less readable than before..

Comment (ledvinap) [inline src/main/mw.c]: Maybe use better name for timestamp comparisons? 
`int32_t timestamp32Diff(int32_t, int32_t)` ?

Comment (tracernz) [inline src/main/mw.c]: Yeah that's better I think. It's a really nice way of handling subtraction of uints producing a negative result, but just hard to come up with a good name for it. :stuck_out_tongue:

**Comment by hydra** [comment]
I'll merge a bunch of these now.

**Comment by hydra** [comment]
See https://github.com/cleanflight/cleanflight/commit/7384e9d4d4b1c2c8657135660cd258b8f95bee9c

I skipped the two commits with comments (see above).  Feel free to update this PR after removing the commits already merged if you think it's still worth merging the other two commits, otherwise you can close this PR.

Thanks again for the cleanups, really appreciate it!

**Comment by hydra** [comment]
great, all merged!

**Pull request closed without merging**